### PR TITLE
add grpc ssl e2e test

### PR DESCRIPTION
**Pull request by qiwzhang**

### CODE REVIEW

File: script/jenkins-utilities [hunk 1]
```diff
@@ -139,13 +139,12 @@ __EOF__
...
 function create_service() {
   local esp_service="${1}"
-  local swagger_json="${2}"
   local output=''
   local operation=''
   local version=''
 
   echo 'Deploying service'
-  retry -n 3 run ${GCLOUD} endpoints services deploy "${swagger_json}"
+  retry -n 3 run ${GCLOUD} endpoints services deploy ${@:2}
```
Comment (JLXIA) [inline script/jenkins-utilities]: is this change necessary?

Comment (qiwzhang) [reply script/jenkins-utilities]: Yes.  to pass multiple config files.   for grpc, you need to pass descriptor file in addition to yaml file
